alexBot/cogs/video_dl.py

```diff
@@ -21,7 +22,7 @@
     re.compile(r'https?://vm\.tiktok\.com/[a-zA-Z0-9#-_!*\(\),]{6,}/'),
     re.compile(r'https?://(?:w{3}\.)tiktok.com/@.*/video/\d{18,20}\??[a-zA-Z0-9#-_!*\(\),]*'),
     re.compile(r'https?://(?:v\.)?redd\.it/[a-zA-Z0-9#-_!*\(\),]{6,}'),
-    re.compile(r'https?://(?:\w{,32}\.)?reddit\.com\/(?:r\/\w+\/)?comments\/[a-zA-Z0-9#-_!*\(\),]{6,}'),
+    re.compile(r'https?://w{0,3}\.?reddit\.com\/(?:r\/\w+\/)?comments\/[\w]+\/\w+\b(?!\/\w)'),
```

Comment:
Revert change to `w{0,3}`, regex was designed to watch some more archaic Reddit subdomain schemes, e.g.:

https://old.reddit.com/r/aww/comments/90bu6w
https://aww.reddit.com/comments/90bu6w